docs/source/content/examples/7-labeling.ipynb

```diff
@@ -59,11 +69,7 @@
    "outputs": [
     {
      "data": {
-      "text/plain": [
-       "{'api_version': '0.2.7',\n",
-       " 'mvg_highest_tested_version': '0.2.7',\n",
-       " 'mvg_version': '0.9.7'}"
-      ]
+      "text/plain": "{'api_version': '0.3.0',\n 'mvg_highest_tested_version': '0.3.0',\n 'mvg_version': '0.12.0'}"
```

Comment:
The version here does not match with the version you have in the mvg.py

Reply:
The version numbering has been updated in the notebook now